lib/syntax_tree/translation/parser.rb

```diff
@@ -1381,7 +1351,8 @@ def visit_if(node)
 
             else_token =
               case node.consequent
-              when Elsif
+              when IfNode
+                # The `IfNode` represents an `elsif` clause
                 source_range_length(node.consequent.location.start_char, 5)
```

Comment:
I think it's correct here to still use the next 5 characters for the `else_token` because the token will still be `elsif`.